test/ApiCheck.Test/ApiListingComparerTests.cs

```diff
@@ -80,6 +79,68 @@ public void Compare_Detects_TypeGenericAritybreakingChanges_as_removal()
             Assert.Contains(expected, breakingChanges);
         }
 
+        [Theory]
+        [InlineData("public class ComparisonScenarios.ClassToRemoveFieldsFrom")]
+        [InlineData("public struct ComparisonScenarios.StructToRemoveFieldsFrom")]
+        public void Compare_DetectsAllFieldRemovals(string typeToCheck)
+        {
+            // Arrange
+            var v1ApiListing = CreateApiListingDocument(V1Assembly);
+            var v2ApiListing = CreateApiListingDocument(V2Assembly);
+            var comparer = new ApiListingComparer(v1ApiListing, v2ApiListing);
+
+            // Oops. The NewInternalProperty addition is a breaking change; makes it impossible to subclass type in
+            // another assembly.
```

Comment:
Sorry I don't understand the comment in the code. What is "oops" about it? And what is the issue?